timescaledb/tutorials/nfl-fantasy-league.md

```diff
@@ -174,19 +174,249 @@ conn.close()
+    'group by gameid, si.team_name, game_date '\
+    '), visiting_games as ( '\
+    'select gameid, plays, s.visitor_team as team_name, s.visitor_score as team_score from play_count p '\
+    'inner join scores s on p.team_name like \'%\' || s.visitor_team || \'%\'  '\
+    'and p.game_date = s."date" '\
+    '), home_games as ( '\
+    'select gameid, plays, s.home_team as team_name , s.home_score as team_score from play_count p '\
+    'inner join scores s on p.team_name like \'%\' || s.home_team || \'%\'  '\
+    'and p.game_date = s."date" '\
+    ') '\
+    'select * from visiting_games '\
+    'union all '\
+    'select * from home_games '\
+    'order by gameid asc, team_score desc'
+
+
+# Import data into a pandas Dataframe from the database using a cursor
+with psycopg2.connect(CONNECTION) as conn:
+        cursor = conn.cursor()
+        cursor.execute(query)
+        data = cursor.fetchall()
+        df = pd.DataFrame(data, columns=['gameid', 'plays', 'team_name', 'team_score'])
+        cursor.close()
+
+# Add column to Dataframe to identify winning and losing team
+a = [1, 0]
+b = a*253
+df.insert(4, "win_bin", b)
+
+# Define font object for title of graph
+font = {'weight': 'heavy',
+        'size': 16,
+        }
+
+# Create a scatterplot of the team's scores vs number of plays
+fig, ax = plt.subplots()
+
+# Split up the data so that wins and loses can be identifyed by color
+scat1 = ax.scatter( x=df['team_score'].loc[df['win_bin'] == 1], y=df['plays'].loc[df['win_bin'] == 1], color='gold', label='Win')
+scat2 = ax.scatter( x=df['team_score'].loc[df['win_bin'] == 0], y=df['plays'].loc[df['win_bin'] == 0], color='rosybrown', label='Loss')
+
+# Specify label and tick information
+ax.set_ylabel('Team\'s Number of Plays for One Game')
+ax.set_xlabel('Team\'s Score for One Game')
+ax.set_title('Outcome of Game Given Scores and Plays ', fontdict=font, pad=20)
+ax.legend(handles=[scat1, scat2], loc='upper right')
+
+# Define figure dimensions
+fig.set_figheight(6)
+fig.set_figwidth(8)
+fig.tight_layout()
+
+plt.show()
+```
+
+Your graph should look like this...
+
+<img class="main-content__illustration" src="" alt="Top Three Players by Position"/>
+
+Let's look at the y-axis, or the number of plays for one team during a single game, it appears that more plays do not always imply a guaranteed win. In fact, the top three teams with the highest number of plays for a single game all appeared to have lost. There are many interesting facts which you could glean from this query, this scatterplot being just one possibility. 
+
+### **Average yards per game for top 3 players of each position**
+
+We will be using the following Postgres query to extract the average yards run by an individual player over one game. This query will only include the top three highest player's average yard values per position type. The data is ordered by the average yards run across all players for each position, later on we will see why this is significant. 
```

Comment:
```suggestion
You can use this PostgreSQL query to extract the average yards run by an individual player over one game. This query will only include the top three highest player's average yard values per position type. The data is ordered by the average yards run across all players for each position. This becomes important later on. 
```

Reply:
I always remind myself: "postgres" when you speak, "PostgreSQL" when you write